src/sentry/api/endpoints/project_group_index.py

```diff
@@ -455,6 +454,10 @@ def put(self, request, project):
                             group=group,
                             reason=GroupSubscriptionReason.status_change,
                         )
+                        if not GroupAssignee.objects.filter(
+                            group=group, project=group.project
+                        ).first():
+                            result['assignedTo'] = (User.objects.filter(id=acting_user.id).first())
```

Comment:
Okay, that makes sense. You can unwrap it with [`extract_lazy_object`](https://github.com/getsentry/sentry/blob/6ba6d960b0e5bcb5e0598f5f8c8f49cba96d3142/src/sentry/utils/functional.py#L12) without having to make a database query here.